core/src/test/java/org/hibernate/ogm/backendtck/dialectinvocations/GridDialectOperationInvocationForOneToOneTest.java

```diff
@@ -14,6 +14,8 @@
 
 import org.hibernate.Session;
 import org.hibernate.Transaction;
+import org.hibernate.ogm.backendtck.associations.onetoone.Husband;
```

Comment:
For info, I'm not a bit fan of tests suing model classes from other tests because when one adds new properties it can destabilize them. Not sure what to do.